src/core/caniuse.js

```diff
@@ -45,22 +46,26 @@ export async function run(conf, doc, cb) {
   if (!caniuse.feature) {
     return; // no feature to show
   }
-  // Opportunistically insert the style into the head to reduce FOUC.
-  document.head.appendChild(hyperHTML `<style>${caniuseCss}</style>`);
+
+  const link = createResourceHint({
+    hint: "preconnect",
+    href: "https://raw.githubusercontent.com",
+  });
+  document.head.appendChild(link);
+  document.head.appendChild(hyperHTML`<style>${caniuseCss}</style>`);
 
   const parent = document.querySelector(".head dl");
   parent.appendChild(hyperHTML`
     <dt class="caniuse-title" id="${`caniuse-${caniuse.feature}`}">
       Can I Use this API?
     </dt>`);
   const placeholder = parent.appendChild(hyperHTML`
-    <dd class="caniuse-stats">fetching data from caniuse.com...</dd>`);
-
-  const cache = new IDBCache("respec-caniuse", ["caniuse"]);
-  await Promise.all([doc.respecIsReady, cache.ready]);
+    <dd class="caniuse-stats"></dd>`);
 
   hyperHTML.bind(placeholder)`${{
-    any: canIUse(caniuse, cache),
+    any: fetchAndCacheJson(caniuse)
+      .then(stats => createTableHTML(caniuse, stats))
+      .catch(err => err),
```

Comment:
we should probably console.error() here, otherwise we won't know there has been an error.

Reply:
the `console.error` are in the `try-catch` blocks already. This `catch` is here just to show the error and fallback (caniuse link) to user.
Should i remove the console.error from there and add here?

Reply:
Ah, does hyperHTML display the error?

Reply:
If yes, then it probably shouldn't. If it errors, we should resolve with the html content, but we shouldn't use `.catch()` here.

Reply:
> Should i remove the console.error from there and add here?

Yes, let's remove from here. I think we can simplify the error handling a little bit more over in `fetchAndCacheJson()`, but will need to think about it.

Reply:
I'll need to move the content of third try-catch into second, unless I'm missing something. The `then-catch` is there just because if the fetch (or `.json()`) fails, we can't `createTableHTML` as it expects `json`. So we show the error to user instead using that `catch`